types/react-avatar-editor/index.d.ts

```diff
@@ -23,6 +24,7 @@ export interface CroppedRect {
 }
 
 export interface AvatarEditorProps {
+    className?: string;
```

Comment:
According to the [react-avatar-edit](https://github.com/mosch/react-avatar-editor) documentation, the `className` can receive the type `String | String[]`. It would be nice to have tests for both cases too.

Reply:
Thanks for taking a look Diogo,

That's not what the code does nor the usual React-way to handle this prop (afaik).
https://github.com/mosch/react-avatar-editor/blob/master/src/index.js#L150
https://github.com/mosch/react-avatar-editor/blob/master/src/index.js#L703

I've opened a ticket for clarification, in the meantime, could we assume the actual implementation is the reference and stick with `string`? 

 https://github.com/mosch/react-avatar-editor/issues/286

Reply:
I just took a look at the component source. Think you are right.